package.json

```diff
@@ -177,6 +177,9 @@
         "@swc/jest"
       ]
     },
+    "globals": {
+      "__DEV__": true
```

Comment:
Does jest read this package.json attribute?

Reply:
If we remove this, we see lots of `ReferenceError: __DEV__ is not defined` errors.